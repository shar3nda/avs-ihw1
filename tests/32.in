10
6 -2 10 -2 10 0 8 6 0 1