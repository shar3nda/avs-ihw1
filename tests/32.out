-2 10 -2 10 0 8 6 0 1